web/cypress/integration/hfi-calculator-page.spec.ts

```diff
@@ -55,7 +55,8 @@ function interceptSetFireStarts() {
 }
 
 function interceptDownload() {
-  cy.intercept('POST', 'api/hfi-calc/download-pdf', {
+  cy.intercept('GET', 'api/hfi-calc/fire_centre/1/2021-08-02/pdf', {
+    // In reality the server returns a pdf file, not a json file - but for this test it really doesn't matter.
```

Comment:
Can you just delete the fixture for this?

Reply:
Right you are. Never needed it.